features/facebookImport/test/importers/name-importer.test.js

```diff
@@ -1,78 +0,0 @@
-"use strict";
```

Comment:
This was no longer relevant?

Reply:
But since it was replaced by `personalDataImporter`, couldn't it be adapted to the new name?